third-party/bootstrap-deps.json

```diff
@@ -194,5 +194,13 @@
         "url": "https://github.com/facebook/gfxreconstruct.git",
         "revision": "99a2ac03e7386c9773804c7b25a3712fd867f458"
     }
+},
+{
+    "name": "nanovg",
+    "source": {
+        "type": "git",
+        "url": "https://github.com/vinsentli/nanovg",

```

Comment:
Can you use the upstream `https://github.com/memononen/nanovg`?

Reply:
The original NanoVG demo relies on OpenGL and GLFW APIs to compile successfully. Since I forked the original NanoVG repository, I made some minor modifications to the demo.c file. 
Here are the changes I made：https://github.com/vinsentli/nanovg/commit/5660d826c3bde672f6d92fde502c06d3029273b7